src/db/scan.rs

```diff
@@ -0,0 +1,502 @@
+use crate::prolly;
+
+#[allow(dead_code)]
+pub struct ScanKey<'a> {
+    value: &'a [u8],
+    exclusive: bool,
+}
+
+#[allow(dead_code)]
+pub struct ScanBound<'a> {
+    // TODO: Make these two fields exclusive?
```

Comment:
I have a preference for only having TODOs for things that we are actually going to do, and soon. Otherwise they just sit there forever. If this doesn't meet the bar maybe just 
```
// These two fields are mutually exclusive.
```
and we're done.

Reply:
Me too. The situation with this code is actually the reverse -- the fields are *not* mutually exclusive currently. I've updated the comment to document the current situation and remove the TODO:

https://github.com/rocicorp/repc/pull/103